src/sc_client/sc_agent.py

```diff
@@ -0,0 +1,46 @@
+"""
+This source file is part of an OSTIS project. For the latest info, see https://github.com/ostis-ai
+Distributed under the MIT License
+(See an accompanying file LICENSE or a copy at http://opensource.org/licenses/MIT)
+"""
+
+import warnings
+from abc import ABC, abstractmethod
+
+from sc_client import client
+from sc_client.models import ScAddr, ScEvent
+from sc_client.sc_keynodes import ScKeynodes
+
+
+class ScAgent(ABC):
```

Comment:
There are tests for this logic?

Reply:
There were no tests even before removing (joke?).

Reply:
I want not to copy tests from kpm

Reply:
Agreed, we will remove this untested logic, so there is no sense to make it now, just waste of time as I see